src/csg/probabilistic_csg.jl

```diff
@@ -36,17 +36,6 @@ function expr2pcsgrammar(ex::Expr)::ContextSensitiveGrammar
 		end
 	end
 	alltypes = collect(keys(bytype))
-	# Normalize probabilities for each type
```

Comment:
Why are we removing normalization of probabilities again?

Reply:
We already do it at the end of this method:

https://github.com/Herb-AI/HerbGrammar.jl/blob/671a3024cf6edfdb78045542b12fa47865f4a22e/src/csg/probabilistic_csg.jl#L46